lib/Doctrine/Annotations/ImplicitlyIgnoredAnnotationNames.php

```diff
@@ -83,10 +83,39 @@ final class ImplicitlyIgnoredAnnotationNames
     ];
 
     private const PHPUnit = [
-        'codeCoverageIgnore'      => true,
-        'codeCoverageIgnoreEnd'   => true,
-        'codeCoverageIgnoreStart' => true,
-        'covers'                  => true,
+        'author'                         => true,
```

Comment:
`@author` is already part of phpDoc itself (see above).

Reply:
Technically it's fine to have duplicates so we can even keep it for completeness.